src/graphnet/training/utils.py

```diff
@@ -115,7 +115,7 @@ def make_train_validation_dataloader(
 ) -> Tuple[DataLoader, DataLoader]:
     """Construct train and test `DataLoader` instances."""
     # Reproducibility
-    rng = np.random.RandomState(seed=seed)
+    rng = np.random.default_rng(seed=seed)
```

Comment:
It looks like the "rng" is passed to `sklearn.model_selection.train_test_split` and `pandas.DataFrame.sample`. Maybe we should just pass the seed value directly to sklearn.model_selection.train_test_split?

Reply:
Do you mean changing `random_state=rng` to `random_state=seed`? If yes, then we can also remove `rng` as it will not be used anywhere.

Reply:
I thought about passing the `np.Generator ` to `pd.DataFrame.sample` and the seed or state to `sklearn.model_selection.train_test_split`, but I don't see any issue passing the seed to both.

Reply:
I apologize for misunderstanding of your comment.